dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -156,10 +150,23 @@ def install(self):
                 raise ImportError(error_msg)
 
         print("Proceed with build...")
-        dali_cflags, dali_lflags = get_dali_build_flags()
-        tf_cflags, tf_lflags = get_tf_build_flags()
         cuda_cflags, cuda_lflags = get_cuda_build_flags()
 
+        # Building a DALI stub library. During runtime, the real libdali.so will be already loaded at the moment when the DALI TF plugin is loaded
+        # This is done to avoid depending on DALI being installed during DALI TF sdist installation
+        dali_stub_src = os.path.join(self.src_path, 'stub', 'dali_stub.cc')
+        dali_stub_lib = 'libdali.so'
```

Comment:
Can we put this stub into some tmp dir or it have to be next to the source code?